kuma/static/img/web-docs-sprite.svg

```diff
@@ -0,0 +1,9 @@
+</svg>
```

Comment:
I don't anticipate adding more than 3 or 4 more SVGs and we only add one or two images a year when we're not redesigning, I'm not sure it's worth updating the build process.